### DISCUSSION TITLE
Rebuild index feature need better logging

### DISCUSSION THREAD

**Discussion by bielu** [umbraco/Umbraco-CMS#16568] [Features and ideas]
Lately i am rebuilding huge indexes in umbraco v13, on top of that umbraco takes ages to rebuild them there is no helpful logging to say if that still goes or it died in background. 
I would expect reindexing to give some type of logs, where it says how many batches index, how many left and how long tooks each of batches ;)

**Comment by nul800sebastiaan** [comment]
Assuming this relates to https://github.com/umbraco/Umbraco-CMS/pull/16546

**Reply by bielu** [reply]
It doesn't directly, the only bit is the batching is shared. This is generally about logs being not helpful the issue in pr is about it being not resilient.
